test/functional/tests.py

```diff
@@ -492,5 +492,28 @@ def test_object_patch_set_data(self):
         self.assertEqual(result,
                 {'Hash': 'QmV4QR7MCBj5VTi6ddHmXPyjWGzbaKEtX2mx7axA5PA13G'})
 
+
```

Comment:
@alexander255 just updated our testing framework to skip the functional tests if an IPFS daemon isn't running. It's a one-line change to make your tests have this feature. Take a look at [this class decorator](https://github.com/ipfs/py-ipfs-api/pull/57/files#diff-322d04fe8e037003724bccda9930e89eR49). If you could just throw that on this test class, I'm good to merge this.